getyourdata/organization/tests.py

```diff
@@ -148,36 +187,22 @@ def test_no_organizations_listed_when_no_organizations_exists(self):
 
     def test_existing_organizations_listed_on_page(self):
         for i in range(0, 5):
-            create_organization(self)
-        verify_all_organizations()
-
-        response = self.client.get(reverse("organization:list_organizations"))
+            create_simple_organization()
 
-        self.assertContains(response, "The Organization", 5)
+        is_certain_amount_of_organizations_listed_on_page(self, 5)
```

Comment:
It's better to turn this into the form self.assertEquals(get_amount_of_organizations_on_page(self), 5).
When skimming the code, it is easier to notice assertations (which start with self.assert) from the rest of the code.